docs/docs/30-how-to-guides/11-working-with-projects.md

```diff
@@ -0,0 +1,113 @@
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Project
+metadata:
+name: kargo-demo
+```
+
+:::note
+Deletion of a `Project` resource results in the deletion of the corresponding
+`Namespace`. For convenience, the inverse is also true -- deletion of a
+project's `Namespace` results in the deletion of the corresponding `Project`
+resource.
+:::
+
+:::info
+There are compelling advantages to using `Project` resources instead of
+permitting users to create `Namespace` resources directly:
+
+* The required label indicating a `Namespace` is a Kargo project cannot be
+forgotten or misapplied.
+
+* Users can be granted permission to indirectly create `Namespace` resources for
+Kargo projects _only_ without being granted more general permissions to create
+_any_ new `Namespace` directly.
+
+* Boilerplate configuration is automatically created at the time of `Project`
+creation. This includes things such as project-level RBAC resources and
+`ServiceAccount` resources.
+:::
+
+## Promotion Policies
+
+A `Project` resource can additionally define project-level configuration. At
+present, this only includes **promotion policies** that describe which `Stage`s
+are eligible for automatic promotion of newly available `Freight`.
+
+:::note
+Promotion policies are defined at the project-level because users with
+permission to update `Stage` resources in a given project `Namespace` may _not_
+have permission to create `Promotion` resources. Defining promotion policies at
+the project-level therefore restricts such users from enabling automatic
+promotions for a `Stage` to which they may lack permission to promote to
+manually. It leaves decisions about eligibility for auto-promotion squarely in
+the hands of someone like a "project admin."
+:::
+
+In the example below, the `test` and `uat` `Stage`s are eligible for automatic
+promotion of newly available `Freight`, but any other `Stage`s in the `Project`
+are not:
+
+```yaml
+apiVersion: kargo.akuity.io/v1alpha1
+kind: Project
+metadata:
+name: kargo-demo
+spec:
+promotionPolicies:
+- stage: test
+autoPromotionEnabled: true
+- stage: uat
+autoPromotionEnabled: true
+```
+
+## Namespace Adoption
+
+At times, `Namespace`s may require specific configuration to
+comply with regulatory or organizational requirements. To
+account for this, Kargo supports the adoption of pre-existing
+`Namespace`s that are labeled with `kargo.akuity.io/project: "true"`.
+This enables you pre-configure such `Namespace`s according to your
+own requirements.
+
+:::info
+`kargo.akuity.io/project: "true"` label serves as a safeguard,
+ensuring that only designated namespaces can be adopted as Kargo `Project`s,
+thereby preventing the inadvertent integration of non-Project namespaces.
+:::
+
+For example, if your policy agent mandates that all `Namespace`s include
+a label identifying the internal organization responsible for them, you
+can define such a `Namespace` directly as in the following example:
```

Comment:
Since this is positioned _after_ the info box, I think it becomes awkward to introduce this with "for example," because the info box has interrupted the flow and the reader may have lost context. "For example of what??"

I think some slight rewording can fix it:

```suggestion
The following example demonstrates adoption of a `Namespace` that's been
pre-configured with with a label unrelated to Kargo:
```